frontend/packages/dev-console/src/components/import/builder/BuilderSection.tsx

```diff
@@ -1,31 +1,30 @@
 import * as React from 'react';
-import { useTranslation } from 'react-i18next';
+import { FormikValues, useFormikContext } from 'formik';
 import { PipelineKind } from '@console/pipelines-plugin/src/types';
 import { NormalizedBuilderImages } from '../../../utils/imagestream-utils';
-import { ImageData } from '../import-types';
 import FormSection from '../section/FormSection';
 import BuilderImageSelector from './BuilderImageSelector';
 import BuilderImageTagSelector from './BuilderImageTagSelector';
 
 export interface ImageSectionProps {
-  image: ImageData;
   builderImages: NormalizedBuilderImages;
   existingPipeline?: PipelineKind;
 }
 
-const BuilderSection: React.FC<ImageSectionProps> = ({
-  image,
-  builderImages,
-  existingPipeline,
-}) => {
-  const { t } = useTranslation();
+const BuilderSection: React.FC<ImageSectionProps> = ({ builderImages, existingPipeline }) => {
+  const {
+    values: {
+      image,
+      import: { showEditImportStrategy },
+    },
+  } = useFormikContext<FormikValues>();
   if (!builderImages) {
     return null;
   }
 
   return (
     <>
-      <FormSection title={t('devconsole~Builder')} fullWidth>
+      <FormSection fullWidth style={!showEditImportStrategy ? { display: 'none' } : {}}>
```

Comment:
same as above. we shouldn't render the FormSection itself instead of setting the style `display: none`

Reply:
We need to add this style because we want to render the component as it sets the required values.

Reply:
The validation should be updated to not require builder image if strategy is not builder image.